### PR TITLE
Updated Englisch language files.

### PR DESCRIPTION
**Pull request by normanjaeckel**

### CODE REVIEW

File: openslides/locale/angular-gettext/template-en.pot [hunk 1]
```diff
@@ -301,146 +627,518 @@ msgstr ""
...
+msgid "Project motion"
+msgstr ""
+
+#: core/static/templates/core/customslide-list.html:34
+#: users/static/templates/users/user-detail.html:12
+#: users/static/templates/users/user-list.html:112
+msgid "Project user"
+msgstr ""
+
+#: core/static/templates/core/projector-controls.html:2
```
Comment (emanuelschuetze) [inline openslides/locale/angular-gettext/template-en.pot]: Please fix in template `core/static/templates/core/projector-controls.html`, line 2: The closing tag has to be a `</h3>` tag. Then the msgstr here should be updated.

**Comment by emanuelschuetze** [comment]
One remark. Rest is ok.

**Comment by normanjaeckel** [comment]
@emanuelschuetze Please review again.

File: openslides/locale/angular-gettext/template-en.pot (hunk 1 above)
Comment (normanjaeckel) [inline openslides/locale/angular-gettext/template-en.pot]: Done.

**Comment by emanuelschuetze** [comment]
Ok, looks good.

**Comment by ostcar** [comment]
ok. Please rebase and merge

**Pull request merged**